app/experimenter/visualization/api/v1/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from experimenter.visualization.api.v1.views import analysis_results_view
+
+urlpatterns = [
+    url(
+        r"^(?P<slug>[\w-]+)/$",
+        analysis_results_view,
+        name="analysis-results-data",
```

Comment:
👍 for adding a descriptive name, but in general the first part of the name should be the name of the "app" it's contained within (ie the name of the folder inside `experimenter/`) which in this case is `visualization`?  Just makes it easier to know where to find the view that implements it.